config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,13 @@
+CarrierWave.configure do |config|
+  config.fog_provider = 'fog/aws'
+
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: ENV['STORAGE_ACCESS_KEY'].to_s,
+    aws_secret_access_key: Secret['STORAGE_SECRET_KEY'].to_s,
+    region: ENV.fetch('STORAGE_REGION', 'us-east-1'),
+    endpoint: ENV['STORAGE_ENDPOINT'],
+  }
+
+  config.fog_directory = ENV.fetch('STORAGE_BUCKET', 'photos')
```

Comment:
AFAIK now in `minio` buckets are created dynamically (each year new bucket will be created, 2017-reports, 2018-reports), which IMO is bad and we might need to consider this during migration.

Reply:
Ok, will need to dig into `minio` . What is current storage structure? Are images currently stored in `minio`/`S3`? Maybe we can keep the same structure not to move assets.

Reply:
Yes, images are stored in minio. Don't know exact structure for it though. I'll check.